src/40_Dynamic_AMP/Dynamic_And_Cached_Data_Details.html

```diff
@@ -0,0 +1,59 @@
+<!-- ## Dynamic and cached product data details -->
+<!--
+This is a sample showing how to implement product data details like description, price, availability etc.
```

Comment:
s/like/such as/

Suggestion on how to rephrase the intro: 

AMPs can be served from the AMP Cache. Due to the AMP Cache's stale-while-invalide caching strategy, one user will see stale data until the latest version has been fetched. This might be a problem if product data needs always to be up-to-date, for example, product availability. This sample demonstrates how to avoid stale data on cached AMPs.

Reply:
done